.github/docker/centreon-web/bookworm/entrypoint/container.sh

```
@@ -7,5 +7,40 @@ set -x
 # ls is required to ensure that the scripts are properly sorted by name.
 BASEDIR="/usr/share/centreon/container.d"
 for file in `ls $BASEDIR` ; do
```

Shellcheck (0.10.0)

[error] Iterating over ls output is fragile. Use globs.

(SC2045)